relay-server/src/actors/processor.rs

```diff
@@ -533,6 +539,8 @@ pub struct EnvelopeProcessorService {
 
 struct InnerProcessor {
     config: Arc<Config>,
+    #[cfg(feature = "processing")]
+    redis_pool: Option<Arc<RedisPool>>,
```

Comment:
Just noticed this now, the `RedisPool` can be cloned and is internally arc'ed. You can safely remove the `Arc`.

Otherwise, the clone below in the constructor would be invalid.

Reply:
like putting reference to redis pool in the evaluator ? It seems I have to use `PhantomData` for that since the lifetimes are only used under redis feature flag